L1Trigger/L1TGlobal/src/GlobalScales.cc

```diff
@@ -169,10 +169,12 @@ long long l1t::GlobalScales::getLUT_CalMuEta(const std::string & lutName, int el
 {
    long long value = 0;
 
-   if(element < (int)m_lut_CalMuEta.find(lutName)->second.size()) {
-     value = m_lut_CalMuEta.find(lutName)->second.at(element);
+   if(element < 0) {
```

Comment:
It seems element should just be an unsigned int. Does something prevent that change?

Otherwise would it not be better t fix this problem at its source instead of covering it up with a printout that will go unnoticed in production?